test/Microsoft.OpenAPI.OData.Reader.Tests/Operation/SingletonGetOperationHandlerTests.cs

```diff
@@ -31,6 +31,7 @@ public void CreateSingletonGetOperationReturnsCorrectOperation(bool enableOperat
         <Annotation Term=""Org.OData.Capabilities.V1.ReadRestrictions"">
           <Record>
             <PropertyValue Property=""Description"" String=""Get the signed-in user."" />            
+            <PropertyValue Property=""LongDescription"" String=""Get the signed-in user."" />            
```

Comment:
Same as previous comment, differentiate the two values.

Reply:
On it.